source/settings/settings-reference.txt

```diff
@@ -107,6 +107,15 @@ You can configure the following settings on the |compass| interface:
        enabled. You cannot toggle this option on Linux. You can toggle this 
        setting only on Windows or macOS.
 
+   * - Show Quit Confirmation Dialog
+     - General
+     - Toggle whether a confirmation dialog is shown when quitting Compass.
```

Comment:
(term and active voice)
I'm a little concerned about toggle. I think we could get more specific. Also we want to stay away from "is shown". So maybe something like:
Specifies whether to display the confirmation dialog when you exit Compass.